### DISCUSSION TITLE
.NET nanoFramework Extension Cannot be packaged

### DISCUSSION THREAD

**Discussion by 28106830** [nanoframework/Home#1574] [Q&A]
### Tool

Visual Studio extension

### Description

In the package of nanoframework project properties, in the package options, set: Generate nuget at build time, even if selected, it will not be generated

### Screenshots

![image](https://github.com/user-attachments/assets/d848fd9f-df30-4c16-a170-ed03cad2bcf0)

**Comment by CoryCharlton** [comment]
Use a [.nuspec](https://github.com/CCSWE-nanoFramework/CCSWE.nanoFramework.Core/blob/master/CCSWE.nanoFramework.Core.nuspec) file and the [nuget cli](https://github.com/CCSWE-nanoFramework/actions-nanoframework/blob/7eecf0c15d28fde7db0bde91ec881fa946e6a974/.github/workflows/build-solution.yml#L58) instead.

**Comment by josesimoes** [comment]
This is not an issue, rather a question. Moving to discussions.

**Comment by 28106830** [comment]
[NanoFrameworkPackageBuilder1.0.8.zip](https://github.com/user-attachments/files/18257145/NanoFrameworkPackageBuilder1.0.8.zip)
I solved it myself. I made a packaging tool myself
As long as the version number is filled in, the packaging can be automatically completed, and it is also possible to configure which files can be added to the package in the bin directory
